src/core/Database.cpp

```diff
@@ -241,20 +241,12 @@ bool Database::save(const QString& filePath, QString* error, bool atomic, bool b
 
             // Delete the original db and move the temp file in place
             QFile::remove(filePath);
-#ifdef Q_OS_LINUX
-            // workaround to make this workaround work, see: https://bugreports.qt.io/browse/QTBUG-64008
-            if (tempFile.copy(filePath)) {
-                // successfully saved database file
-                return true;
-            }
-#else
-            if (tempFile.rename(filePath)) {
+            if (tempFile.QFile::rename(filePath)) {
```

Comment:
A comment why this is necessary might be wise for the next coder touching this stuff?